server/modules/layout_state_machine.js

```diff
@@ -242,22 +242,24 @@ class FadeOutState extends stateMachine.State {
     this.timer_ = null;
   }
   enter(transition) {
-    let deadline = time.inFuture(5000);
+    let now = time.now();
+    const FADE_OUT_DURATION = 5000;
+    let deadline = now + FADE_OUT_DURATION;
     if (monitor.isEnabled()) {
       monitor.update({layout: {
-        time: time.now(),
+        time: now,
         state: this.getName(),
-        deadline: deadline
+        deadline: deadline,
       }});
     }
     
     this.transition_ = transition;
     debug(`Fading out ${this.partition_.length} layouts at ${deadline} ms`);
-    this.partition_.forEach(sm => sm.fadeToBlack(deadline));
+    this.partition_.forEach(sm => sm.fadeToBlack(now));
```

Comment:
Is it a "fade" if it's "now"?

Reply:
The time passed is the moment that the preparation to begin to fade should start. 'deadline' is the moment the fade should start, and yes, this is stupid, and yes, I should fix this structurally, but this gets the fade working again.